utils/installer/install.sh

```diff
@@ -38,6 +38,8 @@ function usage() {
   echo "Options:"
   echo "    -h, --help    Print this help message"
   echo "    -y, --yes     Yes for all choices (Install NodeJS, Python, Rust dependencies)"
+  echo "    -n, --no      No for all choices (Install NodeJS, Python, Rust dependencies)"
```

Comment:
Let's rename these to `--[no]-install-dependencies` where `$INSTALL_DEPDENDENCIES` is set to true by default. I think it would be a lot less confusing for everyone. Then you can call the scripts like so:
`bash install.sh --local --no-install-dependencies`